ui/components/app/create-new-vault/create-new-vault.js

```diff
@@ -126,18 +130,14 @@ export default function CreateNewVault({
       </div>
       {includeTerms ? (
         <div className="create-new-vault__terms">
-          <CheckBox
+          <Checkbox
             id="create-new-vault__terms-checkbox"
             dataTestId="create-new-vault__terms-checkbox"
             checked={termsChecked}
             onClick={toggleTermsCheck}
+            label="create-new-vault__terms-label"
```

Comment:
I think we can move the text of the label to the label prop?

```suggestion
            label={termsOfUse}
```